sdk/ai/azure-ai-resources/setup.py

```diff
@@ -0,0 +1,76 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+import os
+import re
+from io import open
+from typing import Any, Match, cast
+
+import pkg_resources
+from setuptools import find_packages, setup
+
+# Change the PACKAGE_NAME only to change folder and different name
+PACKAGE_NAME = "azure-ai-resources"
+PACKAGE_PPRINT_NAME = "Azure AI control plane SDK"
+
+# a-b-c => a/b/c
+PACKAGE_FOLDER_PATH = PACKAGE_NAME.replace("-", "/")
+# a-b-c => a.b.c
+NAMESPACE_NAME = PACKAGE_NAME.replace("-", ".")
+
+# Version extraction inspired from 'requests'
+with open(os.path.join(PACKAGE_FOLDER_PATH, "_version.py"), "r") as fd:
+    version = cast(Match[Any], re.search(r'^VERSION\s*=\s*[\'"]([^\'"]*)[\'"]', fd.read(), re.MULTILINE)).group(1)
+if not version:
+    raise RuntimeError("Cannot find version information")
+
+with open("README.md", encoding="utf-8") as f:
+    readme = f.read()
+with open("CHANGELOG.md", encoding="utf-8") as f:
+    changelog = f.read()
+
+setup(
+    name=PACKAGE_NAME,
+    version=version,
+    description="Microsoft Azure {} Client Library for Python".format(PACKAGE_PPRINT_NAME),
+    long_description_content_type="text/markdown",
+    long_description=readme + "\n\n" + changelog,
+    license="MIT License",
+    author="Microsoft Corporation",
+    author_email="azuresdkengsysadmins@microsoft.com",
+    url="https://github.com/Azure/azure-sdk-for-python",
+    classifiers=[
+        "Development Status :: 4 - Beta",
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3 :: Only",
```

Comment:
Are we doing something that prevents us from supporting 3.7?

Reply:
As Python 3.7 is EOL and we already gave notice in `azure-ai-ml` (one of this package's dependencies) that support will be deprecated in October 2023, we thought it's best to start with python 3.8

Reply:
Ah gotcha. Should we add `python_requires=">=3.8"` then?

Reply:
We have it on line 66 😄